crates/net/p2p/src/lib.rs

```diff
@@ -52,6 +52,6 @@ pub use headers::client::HeadersClient;
 pub use reputation::{Reputation, ReputationChange, ReputationChangeKind, ReputationChangeWeights};
 
 /// Helper trait that unifies network behaviour needed for fetching blocks.
-pub trait BlockClient: HeadersClient + BodiesClient + Unpin + Clone {}
```

Comment:
we don't want this. please revert.

Reply:
ah because we don't need really to enforce this in too many locations?

Reply:
we have a style already through out the code base of including autotraits as super traits of our traits

Reply:
regardless how often they are actually needed